src/main/resources/ca/corefacility/bioinformatics/irida/database/changesets/22.09/rename-ncbi-export-instruments.xml

```diff
@@ -0,0 +1,109 @@
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="ILLUMINA_GENOME_ANALYZER_II"/>
+            <where>instrument_model='ILLUMINAGAII'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="ILLUMINA_GENOME_ANALYZER_IIX"/>
+            <where>instrument_model='ILLUMINAGAIIX'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="ILLUMINA_GENOME_ANALYZER"/>
+            <where>instrument_model='ILLUMINAGA'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="ILLUMINA_HI_SEQ_1000"/>
+            <where>instrument_model='ILLUMINAHISEQ1000'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="ILLUMINA_HI_SEQ_1500"/>
+            <where>instrument_model='ILLUMINAHISEQ1500'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="ILLUMINA_HI_SEQ_2000"/>
+            <where>instrument_model='ILLUMINAHISEQ2000'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="ILLUMINA_HI_SEQ_2500"/>
+            <where>instrument_model='ILLUMINAHISEQ2500'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="ILLUMINA_HI_SEQ_3000"/>
+            <where>instrument_model='ILLUMINAHISEQ3000'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="ILLUMINA_HI_SEQ_4000"/>
+            <where>instrument_model='ILLUMINAHISEQ4000'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="ION_TORRENT_PGM"/>
+            <where>instrument_model='IONTORRENTPGM'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="ION_TORRENT_PROTON"/>
+            <where>instrument_model='IONTORRENTPROTON'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="LS_454_GS_20"/>
+            <where>instrument_model='ROCHE454GS20'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="LS_454_GS_FLX_TITANIUM"/>
+            <where>instrument_model='ROCHE454GSFLXTI'</where>
+        </update>
```

Comment:
You are missing a migration for `ROCHE454GSFLX`, please add it.

Reply:
Added in d9fb54a4f54ca85a7c06d36b563ff5d431eb22e8